addOns/websocket/src/main/java/org/zaproxy/zap/extension/websocket/ExtensionWebSocket.java

```diff
@@ -381,6 +406,40 @@ public void hook(ExtensionHook extensionHook) {
 
         eventPublisher = new WebSocketEventPublisher(this);
         this.addAllChannelSenderListener(eventPublisher);
+
+        // setup websocket passive scanner
+        if (extensionScript != null
+                && Control.getSingleton()
+                        .getExtensionLoader()
+                        .isExtensionEnabled(ExtensionAlert.NAME)) {
```

Comment:
We should do a null check instead (like with `extensionScript`), this returns the state set by the user in the options (which might change anytime).

Reply:
Fixed in 402252f